README.md

```diff
@@ -1092,6 +1093,18 @@ We will need to install the az cli using Python PIP.
 3. Execute the command "pip install azure-cli" using command line. It takes about 20 minutes to install the azure cli.
 4. Verify the installation using the command az --version.
 
+## Running on Apple Silicon (ARM64)
+
+Running x86 on M1 chip apple devices:
+if having trouble emulating x86 images try installing rosetta via the terminal using 
+  > softwareupdate --install-rosetta
+
+and enabling the following settings in docker desktop under
```

Comment:
Good to mention the minimum docker desktop version needed to achieve this?

Reply:
Added that to it